React/Views/ScrollView/RCTScrollContentView.m

```diff
@@ -51,13 +51,12 @@ - (void)reactSetFrame:(CGRect)frame
   // the contents will overflow causing the scroll indicators to appear unnecessarily.
   NSScrollView *platformScrollView = [scrollView scrollView];
   if ([platformScrollView scrollerStyle] == NSScrollerStyleLegacy) {
-    CGFloat horizontalScrollerHeight = [platformScrollView hasHorizontalScroller] ? NSHeight([[platformScrollView horizontalScroller] frame]) : 0;
+    BOOL contentHasHeight = platformScrollView.contentSize.height > 0;
+    CGFloat horizontalScrollerHeight = ([platformScrollView hasHorizontalScroller] && contentHasHeight) ? NSHeight([[platformScrollView horizontalScroller] frame]) : 0;
     CGFloat verticalScrollerWidth = [platformScrollView hasVerticalScroller] ? NSWidth([[platformScrollView verticalScroller] frame]) : 0;
```

Comment:
we don't want an additional `contentHasWidth` also?

Reply:
reping about this @shwanton

Reply:
I didn't see the same issue for vertical scrollview when the `horizontal` prop was not present. I'm happy to add the check for 0 width as well.

Reply:
If there isn't a bug, I'll merge it as is. Thanks!